egui/src/widgets/plot/mod.rs

```diff
@@ -345,8 +414,9 @@ impl Widget for Plot {
                     transform.zoom(zoom_factor, hover_pos);
                     auto_bounds = false;
                 }
-                let scroll_delta = ui.input().scroll_delta;
+                let mut scroll_delta = ui.input().scroll_delta;
                 if scroll_delta != Vec2::ZERO {
+                    scroll_delta.y *= -1.0;
```

Comment:
Why this change? The scroll direction is correct on my mac - if it is wrong for you, please file a separate issue for that (it should then be fixed for more than just the plot!)

Reply:
Interesting, I didn't think that this would be a platform issue. So on Linux, the two-finger scroll delta reported by `ui.input().scroll_delta` is positive to the right and up. I can enable "natural scrolling" in the settings and this reverses the direction on both axes. If this is different for you I can open an issue.

Reply:
Yeah, it is different for me :( Please open an issue, and we can collect data on what platforms this happens on (native vs web vs os etc)